src/Payments/PaymentMethodsRecord.res

```diff
@@ -527,15 +528,19 @@ let getRequiredFieldsFromJson = dict => {
 
 let dynamicFieldsEnabledPaymentMethods = ["crypto_currency", "debit", "credit", "blik"]
 
-let getPaymentMethodFields = (paymentMethod, requiredFields) => {
+let getPaymentMethodFields = (paymentMethod, requiredFields, isSavedCardFlow) => {
```

Comment:
It would be better to default this value to false right? Making this function with typed params will also help

Reply:
The above code does handle it but the function in itself should treat the isSavedCardFlow as an optional parameter , for example if somebody uses it elsewhere in code.

Reply:
Sure, will add default value to this parameter